public/app/config/config.js

```diff
@@ -26,13 +26,18 @@ const DOWNLOADABLE_MIME_TYPES = [
   'application/pdf',
 ];
 
+// resolve path for windows '\'
```

Comment:
I think we can use some standard function from Node `path`.

https://nodejs.org/api/path.html

Reply:
I thought so as well, but `path.normalize` doesn't change my single `\` to `\\`

Reply:
Do we have any docs about this problem? It seems to me that it is too generic to require a custom solution. Let's talk about this on Friday.